charts/oes/templates/secrets/opsmx-gitops-secret.yaml

```diff
@@ -3,12 +3,19 @@
 apiVersion: v1
 stringData:
   # Repo uri to fetch halyard configuration
+  {{- if (eq .Values.spinnaker.gitopsHalyard.repo.type "git") }}
   gitcloneparam: https://{{ .Values.spinnaker.gitopsHalyard.repo.username }}:{{ .Values.spinnaker.gitopsHalyard.repo.token }}@github.com/{{ .Values.spinnaker.gitopsHalyard.repo.organization }}/{{ .Values.spinnaker.gitopsHalyard.repo.repository }}.git
 
   # Repo details to fetch dynamic configuration
   dynamicaccountsgituri: https://github.com/{{ .Values.spinnaker.gitopsHalyard.repo.organization }}/{{ .Values.spinnaker.gitopsHalyard.repo.dynamicAccRepository }}.git
   gituser: {{ .Values.spinnaker.gitopsHalyard.repo.username }}
   gittoken: {{ .Values.spinnaker.gitopsHalyard.repo.token }}
+  {{- end }}
+  {{- if (eq .Values.spinnaker.gitopsHalyard.repo.type "s3") }}
+  s3accesskey: {{ .Values.spinnaker.gitopsHalyard.repo.s3accesskey }}
+  s3secretkey: {{ .Values.spinnaker.gitopsHalyard.repo.s3secretkey }}
+  s3bucket: {{ .Values.spinnaker.gitopsHalyard.repo.s3bucket }}
```

Comment:
How to specify dynamic accounts repo when repo type is s3? Are we gonna put dynamic accounts as well in s3?